cmd/root.go

```diff
@@ -14,6 +14,7 @@ var (
 	apiVersion      string
 	host            string
 	module          string
+	name			string
```

Comment:
make sure you run `go fmt` on your code before committing or install an editor extension that does it for you!